VersionManager.py

```diff
@@ -64,6 +66,104 @@
                   "tex/gregoriotex-common.tex",
                   "fonts/squarize.py",
                  ]
+COPYRIGHT_FILES = ["install-gtex.sh",
+                   "tex/gregoriotex-signs.tex",
+                   "tex/gregorio-vowels.dat",
+                   "tex/gsp-default.tex",
+                   "tex/gregoriotex-nabc.lua",
+                   "tex/gregoriotex-symbols.lua",
+                   "tex/gregoriotex-chars.tex",
+                   "tex/gregoriotex-main.tex",
+                   "tex/gregoriotex-spaces.tex",
+                   "tex/Makefile.am",
+                   "tex/gregoriotex-common.tex",
+                   "tex/gregoriotex-syllable.tex",
+                   "tex/gregoriotex.lua",
+                   "tex/gregoriotex.sty",
+                   "tex/gregoriosyms.sty",
+                   "tex/gregoriotex-nabc.tex",
+                   "tex/gregoriotex.tex",
+                   "tex/gregoriotex-signs.lua",
+                   "tex/gregoriotex-symbols.tex",
+                   "contrib/TeXShop/Makefile.am",
+                   "contrib/900_gregorio.xml",
+                   "contrib/Makefile.am",
+                   "contrib/gabc.lang",
+                   "contrib/gprocess",
+                   "contrib/checkSyllabation.py",
+                   "configure.ac",
+                   "examples/Makefile.am",
+                   "Makefile.am",
+                   "install.sh",
+                   "debian/copyright",
+                   "debian/copyright",
```

Comment:
Any reason to have this repeated?

Reply:
When I made the initial list of files grep found two locations in this file and returned the filename twice. I did not scan the list for duplicates. Removed.